tailwind.config.js

```diff
@@ -35,6 +36,11 @@ module.exports = {
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
         },
     },
+    fontSize: {
+      '2lg': ['2rem',{
+        lineHeight: '2.4rem'
+      } ],
+      },
```

Comment:
To keep it readable, I would suggest:
```js
fontSize: {
  '2lg': ['2rem', { lineHeight: '2.4rem' }],
},
```
So, you can add more rules if needed below this rule with the same format.